catchpoint-mixin/panels.libsonnet

```diff
@@ -9,6 +9,110 @@ local utils = commonlib.utils;
       local stat = g.panel.stat,
       local barGauge = g.panel.barGauge,
       local fieldOverride = g.panel.table.fieldOverride,
+      local alertList = g.panel.alertList,
+      topAvgLoadTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average total load time by test name',
+          targets=[t.topAvgLoadTimeTestName],
+          description='Average total time it took to load the webpage by test name, helping assess the responsiveness and speed of the monitored webpage.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
 
+      topMaxTotalLoadTime:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top max total load time',
+          targets=[t.topMaxTotalLoadTime],
+          description='Maximum total time it took to load the webpage by test name, indicating the worst-case scenario for webpage loading performance.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgDocumentCompletionTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average document completion time by test name',
+          targets=[t.topAvgDocumentCompletionTimeTestName],
+          description='Average time taken for the browser to fully render the page after all resources are downloaded by test name, serving as a key indicator of end-user perceived load time.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topMaxDocumentCompletionTime:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top max document completion time',
+          targets=[t.topMaxDocumentCompletionTime],
+          description='Maximum time taken for the browser to fully render the page after all resources are downloaded by test name, highlighting the worst-case scenario for end-user perceived load time.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgRequestRatioTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average request ratio by test name',
```

Comment:
Can update this to Bottom like with couchbase: https://github.com/grafana/jsonnet-libs/blob/60f98f603ffb2f88b360d9e31dafa5b0422914a7/couchbase-mixin/dashboards/couchbase-bucket-overview.libsonnet#L476